Strings.props

```diff
@@ -4,6 +4,9 @@
     <PrepareForBuildDependsOn>
       UpdateStringSources;UpdateStringsHeader;$(PrepareForBuildDependsOn)
     </PrepareForBuildDependsOn>
+    <ComputeCompileInputsTargets>
```

Comment:
this is just to hide the glut of generated string sources in the cswinrt project node (the previous technique stopped working with a VS update)